bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -190,7 +190,18 @@ export default function Whiteboard(props) {
               persistShape(shape, whiteboardId);
             }
           });
-    removeShapes(deletedShapes, whiteboardId);
+
+    //order the ids of shapes being deleted to prevent crash when removing a group shape before its children
+    const orderedDeletedShapes = [];
+    deletedShapes.forEach(eid => {
+      if (shapes[eid]?.type !== 'group') {
+        orderedDeletedShapes.unshift(eid);
+      } else {
+        orderedDeletedShapes.push(eid)

```

Comment:
I think it has to be the reverse here, first delete the group shape, and then the children

Reply:
deleting the group shape first leads to the following error:

![Screenshot from 2023-01-20 16-51-45](https://user-images.githubusercontent.com/3728706/213793070-91fff3f9-12a3-42cb-bf66-c5f2b2435af0.png)